tests/test_io.py

```diff
@@ -1,8 +1,11 @@
 import numpy as np
+import os
+from pytest import mark
 
 from kpconv_torch.io import las, ply, xyz
 
 
+@mark.dependency()
```

Comment:
Could you explain the need for `pytest-dependency` there, or alternatively provide a link to the relevant documentation? (The usefullness of an empty dependency declaration is not clear to me.)

Reply:
I believed it was necessary to mark the test representing the dependency with and empty declaration, since you did that in [`test_train.py`](https://github.com/delhomer/kpconv-torch/blob/21a1aabf508d24d68bb9d548e54f5cdae7463369/tests/test_train.py#L51). But it seems indeed useless, so, I remove it.

Reply:
Here is an example : https://pytest-dependency.readthedocs.io/en/stable/usage.html

Reply:
The dependency is useful, since reading applies on the files written by the first group of tests.

Reply:
I did not remember, sorry. Ok for the idea, and thanks for the reference!